F2.2 improvements

ElectricRCAircraftGuy opened the pull request:
F2.2: Improve clarity & wording
    
Resolves this issue: https://github.com/KiCad/kicad-website/issues/318
And this issue: https://github.com/KiCad/kicad-website/issues/319

Ignore the first commit description below. The commits afterwards  undid it. Idid NOT end up adding any such "B" to the example. I was confused. See this old pull request here for info: https://github.com/KiCad/kicad-website/pull/323.

This pull request is ready to be accepted.

marekr commented:
@ElectricRCAircraftGuy For the record, you can update existing pull requests instead of opening a new one each time. Its as simple as just pushing to the same branch on your fork you made the changes.

ElectricRCAircraftGuy commented:
Yeah I'm still trying to figure out how that work flow works. The first time, the problem was that I amended a commit with new info, so `git push` didn't want to accept it now since I modified the previous commit hash. The 2nd time, I renamed the branch since the first branch name was no longer valid, so that messed stuff up. Not sure how to work out these details exactly...

marekr commented:
When you amend a commit, you alter the history. Git pushes will be rejected if history is altered. So you must be git push --force which tells it to disregard the fact the history change.

The 2nd case unfortunately you can't edit the  branch name for the pull request it was looking at when you made it. However you can always do a git push to the old name and github will pick it up again.

ElectricRCAircraftGuy commented:
Thanks I'll keep that in mind.

SchrodingersGat commented:
@marekr these changes all look fine to me if you want to merge :)
